administrator/language/en-GB/com_guidedtours.ini

```
@@ -0,0 +1,98 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_GUIDEDTOURS="Guided Tours"
+COM_GUIDEDTOURS_ACCESS_GUEST="Guest"
+COM_GUIDEDTOURS_ACCESS_PUBLIC="Public"
+COM_GUIDEDTOURS_ACCESS_REGISTERED="Registered"
+COM_GUIDEDTOURS_ACCESS_SPECIAL="Special"
+COM_GUIDEDTOURS_ACCESS_SUPER_USERS="Super users"
+COM_GUIDEDTOURS_BASIC_STEP="Basic Step"
+COM_GUIDEDTOURS_DESCRIPTION="Description"
+COM_GUIDEDTOURS_DESCRIPTION_ASC="Description ascending" 
```

```suggestion
COM_GUIDEDTOURS_DESCRIPTION_ASC="Description ascending"
```

removes extra space at end of line which was reported as an error when language debug is enabled